### PR TITLE
[CFM-961] notify on content recommendation

### PR DESCRIPTION
**Pull request by anvmn**
## This PR is dependent on #1059 

https://issuetracker.amplexor.com/jira/browse/CFM-961

### CODE REVIEW

**Comment by anvmn** [comment]
out

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module [hunk 1]
```diff
@@ -76,6 +77,19 @@ function _c4m_message_format_welcome_to_group_from_field(&$message, $message_ent
 }
 
 /**
+ * Formats 'c4m_notify_content_recommendation' message From field, per template.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module]: @anvmn 
Can you please write the name of the template ?
thanks.

**Comment by anvmn** [comment]
@ordavidil 
This notification contains the 'unsubscribe' link, which implementation (CFM 1187) may change, once Kevin clarifies CFM 470.
Changes at https://github.com/capacity4dev/capacity4more/pull/923 (PR of CFM 1187), will require changes here.

**Comment by ordavidil** [comment]
Assign it to me when it's done then.

**Comment by soniCaH** [comment]
Subscribed users do not get notifications on recommendations.

recnotifications

**Comment by anvmn** [comment]
## Replaced by #1189

**Pull request closed without merging**